chain/types/ethtypes/eth_types.go

```diff
@@ -156,18 +156,19 @@ type EthBlock struct {
 }
 
 var (
-	EmptyEthBloom = [256]byte{}
-	EmptyEthHash  = EthHash{}
-	NoUncleHash   = must.One(ParseEthHash("0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"))
-	EmptyEthInt   = EthUint64(0)
-	EmptyEthNonce = [8]byte{0, 0, 0, 0, 0, 0, 0, 0}
+	EmptyEthBloom  = [256]byte{}
+	EmptyEthHash   = EthHash{}
+	EmptyUncleHash = must.One(ParseEthHash("0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"))
```

Comment:
@magik6k can you add a comment for these 2 lines, explaining what these magic values are?